examples/knowledge_tuning/translation_example/create_seed_data.py

```
@@ -0,0 +1,68 @@
+import os
+import sys
+from tqdm import tqdm
+from datasets import Dataset, load_dataset
+
+sys.path.insert(0, os.path.abspath(os.path.join(os.getcwd(), "..")))
+from knowledge_utils import chunk_document
```

Flake8 (7.2.0)

[error] module level import not at top of file

(E402)